### DISCUSSION TITLE
Welcome to todo-list Discussions!

### DISCUSSION THREAD

**Discussion by Juniirr** [Juniirr/todo-list#7] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ThaliaTambraico** [comment]
Prueba final todo lis finalizado

**Reply by Juniirr** [reply]
esta bien , todo correcto

**Comment by Juniirr** [comment]
Compañeras el home esta fallando ,podrían revisarlo por favor

**Reply by ThaliaTambraico** [reply]
Está bien, lo revisarémos

**Reply by Karla876lll** [reply]
Esta bien compañero
